docs/OpenCLCOnVulkan.md

```diff
@@ -290,7 +291,7 @@ argument as `pod_ubo` rather than the default of `pod`.
 
 Normally plain-old-data arguments are passed into the kernel via a storage buffer.
 Use the option `-pod-pushconstant` to pass these parameters in via push constants. The
-option `-cluster-pod-kernel-args` must also be specified. Push constants are intended
+option `-cluster-pod-kernel-args=0` cannot be specified. Push constants are intended
```

Comment:
Weird. That's how the LLVM command line options work?
Would have expected -no-cluster-pod-kernel-args
But hey, it's documented, so that's fine with me.